### DISCUSSION TITLE
Should prefix be _ or m_ / s_ based on field type

### DISCUSSION THREAD

**Discussion by mfhum** [Humteria/humteria-backend#25] [General]
@baltermia: I add a m_ prefix for member fields and s_ for static fields. I guess we can just use the underscore if you've used that everywhere else, I haven't really looked closer into it when writing the code.

**Comment by mfhum** [comment]
Me Personally havent used it yet and dont quite know what "member" is in this relation

**Reply by baltermia** [reply]
`member` in itself doesn't mean alot, it just says that something is a (I think) non-static "feature" of a class/struct ([members](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/members)).
you can read about fields here: [fields](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/fields)
fields are mostly used in private as backing fields for public [properties](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties). 
I can recommend you reading through the first article above (and the sub-articles) to get a grasp of what those terms mean in .net since it's pretty useful.
